### PR TITLE
Adding Initial State, Dispatch Actions, and Book Manipulation

### PR DESCRIPTION
**Pull request by kessie2862**
This PR includes the following changes to enhance the functionality of the Bookstore project.

- [ ] Added an `initialState` variable to `bookSlice` to provide the initial set of books. The `initialState` includes three book objects with properties such as item_id, title, author, and category.

- [ ] Implemented a reusable component to display the books received from the bookSlice. This component will be responsible for rendering the books dynamically.

- [ ] Utilized the useDispatch hook to dispatch actions within the React project. This enables the manipulation of the book data.

- [ ] Added a `Button` component that includes an event handler to add a book to the books array. The book is added with attributes such as id, title, and author.

- [ ] Implemented a `Button` component that includes an event handler to remove a book from the books array. The book is identified by its id.

### CODE REVIEW

File: src/redux/bookSlice.js [hunk 1]
```diff
@@ -1,11 +1,35 @@
...
   name: 'books',
-  initialState: [],
+  initialState,
   reducers: {
-    addBook: (state, action) => [...state, action.payload],
-    removeBook: (state, action) => state.filter((book) => book.id !== action.payload),
+    addBook: (state, action) => {
+      state.push(action.payload);
+    },
+    // removeBook: (state, action) => state.filter((book) => book.item_id !== action.payload),
```
Comment (binhussen) [inline src/redux/bookSlice.js]: - Kindly, Do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.  👍

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @kessie2862 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- Used Redux in React components ✔️
- Well Structured Code ✔️
- Used Git flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/bookSlice.js (hunk 1 above)
Comment (kessie2862) [reply src/redux/bookSlice.js]: Removed. Thanks

**Review by Davidosky007 — APPROVED** [review]
Hi @kessie2862 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* Redux setup properly 👍 
* Store, Actions and Reducers used in React properly 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**